system/assets/hypha-image-uploader.js

```diff
@@ -0,0 +1,55 @@
+$(function() {
+	$('[data-image-uploader]').each(function () {
+		// get input
+		var $input = $(this);
+		console.log($input.data('image-uploader-src'));
+
+		var src = $input.data('image-uploader-src');
+
+		// Add preview
+		var $preview;
+
+		$input.on('change', function () {
+			createPreviewImage(window.URL.createObjectURL(this.files[0]));
+			toggleInputAndDelete();
+		});
+		var createPreviewImage = function (src) {
+			if (src) {
+				$preview = $('<img class="preview_uploaded_image" src="' + src + '">').insertAfter($input);
+				$('<a href="#"><img src="https://image.flaticon.com/icons/svg/261/261935.svg" style="width:40px;height:40px;" accept="image/png,image/jpeg,image/jpg,image/gif,image/bmp"></a>')
```

Comment:
What's the point of the `accept` attribute here? I could not find any info on it?

Also, this link and/or img should probably get a CSS class and the width/height should be in hypha.css.

Reply:
placed the accept at the input field, where it belongs.

Reply:
moved styling to css